config/software/perl-thread-queue.rb

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2019 Chef Software, Inc.
```

Comment:
Feel free to put your own copyright / author tags here. This is your work not ours